guides/cookbook.md

```diff
@@ -1404,6 +1810,39 @@ symbol table that begins the stream.)
 Not currently supported.
 </div>
 
+<div class="tabpane Python" markdown="1">
+Shared symbol tables may be constructed via the
+[shared_symbol_table](https://ion-python.readthedocs.io/en/latest/amazon.ion.html#amazon.ion.symbols.shared_symbol_table)
+function.
+
+```python
+from amazon.ion.symbols import shared_symbol_table
+
+table = shared_symbol_table(u'test.csv.columns', 1, (u'id', u'type', u'state'))
+```
+
+Reusing the `structs` sequence from the previous Python example, the data may be
```

Comment:
if we can add a reference to the "previous Python example" here that would be best.

Reply:
Reproduced the code from the previous example inline.